dashboard/app/models/script.rb

```diff
@@ -459,6 +460,35 @@ def self.add_script(options, data)
 
     script.stages = script_stages
     script.reload.stages
+
+    # Generate PLC objects
+    if script.professional_learning_course
+      course = Plc::Course.find_or_create_by! name: 'Default'
```

Comment:
This will be used by real courses.  Alternatively I can add the following to plc-cspunit3.script and `find_or_create_by!` the associated `Plc::Course`:
```
plc_course_name "Teacher Support for Computer Science Principles"
```